diem-move/diem-framework/experimental/demo-cli/bars_account.key

```diff
@@ -0,0 +1 @@
+ <�A}_/,��@f��#;P�d�3Ϸ��fh-�
```

Comment:
Is this just for testing or is it a legitimate key?  Otherwise, I'd find a different place to put it.